docs/interop-javapoet.md

```diff
@@ -0,0 +1,53 @@
+KotlinPoet-javapoet
```

Comment:
nit: same as with KSP, maybe `JavaPoet extensions for KotlinPoet`?

Reply:
```suggestion
JavaPoet Extensions for KotlinPoet
```